app/views/sections/_form.html.erb

```diff
@@ -1,79 +1,141 @@
-        <p class="help-block">This manuals page will not have any collapsible sections. The formatting markdown ## will create a top level heading (H2) only, not a collapsible section.</p>
+      <div class="add-vertical-margins">
+        <input name="section[visually_expanded]" type="hidden" value="0" autocomplete="off">
+        <%= render "govuk_publishing_components/components/checkboxes", {
+          name: "section[visually_expanded]",
+          id: "section_visually_expanded",
+          items: [
+            {
+              label: "Remove collapsible content functionality (accordions)",
+              value: 1,
+              checked: section.visually_expanded
+            }
+          ]
+        } %>
+        <%= render "govuk_publishing_components/components/inset_text", {
+          text: "This manuals page will not have any collapsible sections. The formatting markdown ## will create a top level heading (H2) only, not a collapsible section."
+        } %>
       </div>
 
-      <% if manual.has_ever_been_published? && section.accepts_minor_updates? %>
-        <div class="checkbox add-vertical-margins">
-          <%= f.radio_button :minor_update, 1, class: 'js-update-type-minor', tag_type: :p, label: 'Minor update', checked: section.minor_update %>
-          <p class="help-block">Only use for minor changes like fixes to typos, links, GOV.UK style or metadata.</p>
-        </div>
-        <div class="checkbox add-vertical-margins">
-          <%= f.radio_button :minor_update, 0, class: 'js-update-type-major', tag_type: :p, label: 'Major update', checked: !section.minor_update %>
-        </div>
-        <div class="js-change-note-container <%= section.minor_update ? 'js-hidden' : nil %>">
-          <%= f.text_area :change_note, rows: 20, cols: 40, class: 'form-control short-textarea' %>
-          <p class="help-block">This will be publically viewable on GOV.UK.</p>
-        </div>
-      <% else %>
-        <div>
-          <p>Major update</p>
-          <p class="help-block">Adding a new section is always a major update</p>
-          <%= f.text_area :change_note, rows: 20, cols: 40, class: 'form-control short-textarea' %>
-          <p class="help-block">This will be publically viewable on GOV.UK.</p>
-        </div>
-      <% end %>
-
-      <div class="actions">
-        <button name='draft' class="btn btn-success" data-disable-with="Saving...">Save as draft</button>
+      <div>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Change note",
+          heading_level: 2,
+          font_size: "l",
+          margin_bottom: 3,
+        } %>
+        <% if manual.has_ever_been_published? && section.accepts_minor_updates? %>
+          <%= render "govuk_publishing_components/components/radio", {
+            name: "section[minor_update]",
+            items: [
+              {
+                text: "Major update",
```

Comment:
Should this be "Minor"?

Reply:
Oh I see, would have been easier to understand if the field had been called "update_type"